grace/io/image_dataset.py

```diff
@@ -28,20 +28,22 @@ class ImageGraphDataset(Dataset):
         Function to read images from image filenames
     transform : Callable
         Transformation added to the images and targets
+    image_filetype : str
+        File extension of the image files
     """
 
     def __init__(
         self,
         image_dir: os.PathLike,
         grace_dir: os.PathLike,
-        image_reader_fn: Callable,
         *,
         transform: Callable = lambda x, g: (x, g),
+        image_filetype: str = "mrc",
     ) -> None:
-        self.image_reader_fn = image_reader_fn
+        self.image_reader_fn = FILETYPES[image_filetype]
         self.transform = transform
 
-        image_paths = list(Path(image_dir).iterdir())
+        image_paths = list(Path(image_dir).glob(f"*.{image_filetype}"))
```

Comment:
what if there's no images of that filetype?

Reply:
@mooniean The code fails then. The user should input the filetype expected.